lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -776,6 +797,100 @@
             }
           }
 
+          .xseries-action{
+            .xseries-msg{
+              @include float(left);
+              width: flex-grid(9, 12);
+            }
+
+            .message-copy{
+              @extend %t-demi-strong;
+              margin-top: 0;
+            }
+
+            .message-copy-bold{
+              @extend %t-strong;
+            }
+
+            .xseries-border-btn {
+              @extend %btn-pl-black-border;
+              @include float(right);
+              position: relative;
+              left: 10px;
+              padding: ($baseline*0.4) ($baseline*0.6);
+              background-image: none ;
+              text-shadow: none;
+              box-shadow: none;
+              text-transform: none;
+
+              .action-xseries-icon{
+                @include float(left);
+                display: inline;
+
+                @include margin-right($baseline*0.4);
+                background: url('#{$static-path}/images/icon-sm-xseries-black.png') no-repeat;
+                background-color: transparent;
+
+                width: ($baseline*1.1);
+                height: ($baseline*1.1);
+              }
+              &:hover,
+              &:focus {
+
+                .action-xseries-icon{
+                  @include float(left);
+                  display: inline;
+
+                  @include margin-right($baseline*0.4);
+                  background: url('#{$static-path}/images/icon-sm-xseries-white.png') no-repeat;
+                  background-color: transparent;
+
+                  width: ($baseline*1.1);
+                  height: ($baseline*1.1);
+                }
+              }
+            }
+
+            .xseries-base-btn {
+              @extend %btn-pl-black-base;
+              @include float(right);
+              position: relative;
+              left: 10px;
+              padding: ($baseline*0.4) ($baseline*0.6);
+              background-image: none ;
+              text-shadow: none;
+              box-shadow: none;
+              text-transform: none;
+
+              .action-xseries-icon{
+                @include float(left);
+                display: inline;
+
+                @include margin-right($baseline*0.4);
+                background: url('#{$static-path}/images/icon-sm-xseries-white.png') no-repeat;
```

Comment:
These icons are still fuzzy. @marcotuts you can see what I'm talking about on @jimabramson's sandbox. Are you comfortable proceeding with these?